### PR TITLE
BigAnimal: high availability cluster types name change

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

https://enterprisedb.atlassian.net/browse/UPM-23067
https://enterprisedb.atlassian.net/browse/UPM-22668

Updated names and links except for Managing clusters CLI topic.

### CODE REVIEW

File: product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@ PostgreSQL clusters in BigAnimal are continuously backed up through a combinatio
...
+With faraway replicas, the primary server writes to the archive, which is moved to the object store. The replica reads from the object store as files arrive. Prior to the archive replicating across to the replica, the log file must write at least 16MB (the WAL segment size) of data to the WAL to cut a new archive file. If the time interval for closing a WAL file is too large, it can introduce a delay until the replica receives the latest record. The amount of time that it takes to fill up the 16MB log file and copy it to the archive is the replication lag. The replication lag is the period of time when data can be lost. It can be minutes or hours, depending on the amount of activity in your database. You need to measure the replication lag to determine if it's acceptable from a possible data-loss perspective. If it isn't acceptable, consider using a dedicated-high-availability cluster.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx:23]: Why is this dedicated and not distributed?

File: product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx [hunk 2]
```diff
@@ -36,7 +36,7 @@ BigAnimal retains backups for 30 days by default. You can restore backups into a
 
 The restore operation is available for any cluster that has at least one available backup. For newly created clusters, the initial backup becomes available a few minutes after the new cluster is fully operational.
 
-#### Single-node or high-availability clusters restore
+#### Single-node or primary/standby high-availability clusters restore
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx:39]: Can we change this and the next one to Restoring xxx clusters?

File: product_docs/docs/biganimal/release/overview/02_high_availability.mdx [hunk 3]
```diff
@@ -6,21 +6,24 @@ redirects:
...
 |    Postgres distribution     | Versions |          Cluster type          |     Deployment type     |
 | ---------------------------- | -------- | ------------------------------ | ----------------------- |
-| PostgreSQL                   | 11–15    | Single node, high availability | Your cloud account  |
-| PostgreSQL                   | 12–15    | Single node, high availability | BigAnimal cloud account  |
-| EDB Postgres Advanced Server | 12–15    | Single node, high availability | Your cloud account  |
-| EDB Postgres Advanced Server | 14–15    | Single node, high availability | BigAnimal cloud account |
-| EDB Postgres Advanced Server | 14–15    | Extreme high availability      | Your cloud account  |
-| EDB Postgres Extended Server | 14-15    | Extreme high availability      | Your cloud account  |
+| PostgreSQL                   | 11–15    | Single-node, primary/standby high-availability | Your cloud account  |
+| PostgreSQL                   | 12–15    | Single-node,
```
[the remark this answers is not in the document]
Comment (drothery-edb) [reply product_docs/docs/biganimal/release/overview/02_high_availability.mdx:19]: primary/standby high-availability is the name of the cluster type

File: product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx (hunk 1 above)
Comment (drothery-edb) [reply product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx]: typo

File: product_docs/docs/biganimal/release/overview/02_high_availability.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (ebgitelman) [reply product_docs/docs/biganimal/release/overview/02_high_availability.mdx:19]: Right, but if you look at the new version of the text, there are rows that look like they are missing text

File: product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx (hunk 2 above)
Comment (drothery-edb) [reply product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx:39]: might be easier to merge this PR and then fix the headings in this topic. I've made some major changes to it in another [PR](https://github.com/EnterpriseDB/docs/pull/4584).

Comment (drothery-edb) [reply product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx]: done

**Comment by drothery-edb** [comment]
Incorporated Natalia's comments provided in Slack

**Pull request merged**